src/services/identity/DeploymentClient.ts

```diff
@@ -13,6 +13,9 @@ import {
   ListJobsCommand,
```

Comment:
does this mean we need to re-run this on all the quickiefied apps so far? they don't have redirects present?

Reply:
yes this would be the case, will be looking at all the apps for the past mth to change redirect rules